qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -153,17 +154,11 @@ def run_test (self):
         assert_equal("Invalid parameter, spending key for address does not belong to wallet" in errorString, True)
 
         # Verify that debug=zrpcunsafe logs params, and that full txid is associated with opid
-        logpath = self.options.tmpdir+"/node0/regtest/debug.log"
         logcounter = 0
-        with open(logpath, "r") as myfile:
-            logdata = myfile.readlines()
-        for logline in logdata:
-            if myopid + ": z_sendmany initialized" in logline and mytaddr in logline and myzaddr in logline:
-                assert_equal(logcounter, 0) # verify order of log messages
-                logcounter = logcounter + 1
-            if myopid + ": z_sendmany finished" in logline and mytxid in logline:
-                assert_equal(logcounter, 1)
-                logcounter = logcounter + 1
+        check_node_log(self, 0, myopid + ": z_sendmany initialized", False)
+        logcounter = logcounter + 1
+        check_node_log(self, 0, myopid + ": z_sendmany finished", False)
+        logcounter = logcounter + 1
         assert_equal(logcounter, 2)
```

Comment:
This doesn't assert the same thing as the original test; the original would fail if `z_sendmany initialized` and `z_sendmany finished` appeared in the opposite order.

```suggestion
        initialized_line = check_node_log(self, 0, myopid + ": z_sendmany initialized", False)
        finished_line = check_node_log(self, 0, myopid + ": z_sendmany finished", False)
        assert(initialized_line < finished_line)
```